lib/active_fulfillment/services/shopify_api.rb

```diff
@@ -115,9 +118,9 @@ def parse_response(response, root, type, key, value)
 
     def encode_payload(payload, root)
       case @format
-      when 'json'
+      when JSON_DATATYPE
         {root => payload}.to_json
-      when 'xml'
+      when XML_DATATYPE
         payload.to_xml(:root => root)
```

Comment:
Agreed, see my comments in https://github.com/Shopify/active_fulfillment/pull/71. We should really only be creating constants if they're used in more than one place, otherwise just freezing strings inline is my preference.